app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/removepassword/RemovePasswordViewModel.kt

```diff
@@ -0,0 +1,98 @@
+package com.x8bit.bitwarden.ui.auth.feature.removepassword
+
+import android.os.Parcelable
+import androidx.lifecycle.SavedStateHandle
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.data.auth.repository.AuthRepository
+import com.x8bit.bitwarden.ui.platform.base.BaseViewModel
+import com.x8bit.bitwarden.ui.platform.base.util.Text
+import com.x8bit.bitwarden.ui.platform.base.util.asText
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.update
+import kotlinx.parcelize.Parcelize
+import javax.inject.Inject
+
+private const val KEY_STATE = "state"
+
+/**
+ * Manages application state for the Set Password screen.
+ */
+@HiltViewModel
+class RemovePasswordViewModel @Inject constructor(
+    private val authRepository: AuthRepository,
+    savedStateHandle: SavedStateHandle,
+) : BaseViewModel<RemovePasswordState, RemovePasswordEvent, RemovePasswordAction>(
+    initialState = savedStateHandle[KEY_STATE] ?: run {
+        val orgName = authRepository.userStateFlow.value
+            ?.activeAccount
+            ?.organizations
+            ?.firstOrNull { it.shouldUseKeyConnector }
```

Comment:
I feel like it would be a wild edge case, but could a user ever be part of two orgs?

Reply:
Technically yes but I don't think we support multiple key connectors